src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -23,8 +23,10 @@ public DisputeController(IDisputeService disputeService, ILogger<DisputeControll
     }
 
     /// <summary>
-    /// Returns all Disputes from the Oracle Data API.
+    /// Returns all Disputes from the Oracle Data API based on a specified type to exclude and older than date.
     /// </summary>
+    /// <param name="olderThan"></param>
+    /// <param name="status"></param>

```

Comment:
Fill comment here to make it clear the param is an exclusion param, not a filterby param.

Reply:
Done.